mmv1/templates/terraform/examples/sql_mysql_instance_private_ip.tf.erb

```diff
@@ -1,47 +0,0 @@
-# [START cloud_sql_mysql_instance_private_ip]
```

Comment:
Hmm, I'm not sure exactly what's going on with these removals. Are these unused?

Reply:
I did the search and don't find their usage.